src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -94,6 +94,7 @@
                     + "                                         ERROR,WARN,INFO,DEBUG,TRACE [default: INFO].\n"
                     + " -Q, --quickcheck-test-mode              Run in QuickCheck test mode\n"
                     + " -P <seconds>, --cm-poll-interval        Configuration manager poll interval [default: 1]\n"
+                    + " -v, --verify                            Compute and verify checksums for log entries.\n"
```

Comment:
Ok, that makes sense.

I'm going to do the following change:
I will add initialization code in the logunit where the logs directory is scanned on startup. During this scan, all log files are read and the a checksum would be computed for each entry, if the -no-verify flag isn't set. If the -no-verify flag is set, then the log files will still be scanned to check that their header has a type 0 field.